wafer/schedule/tests/test_schedule_editor.py

```diff
@@ -133,10 +136,15 @@ def _start(self):
     def test_access_schedule_editor_no_login(self):
         """Test that the schedule editor isn't accessible if not logged in"""
         self.driver.get(self.edit_page)
-        header = WebDriverWait(self.driver, SELENIUM_WAIT_TIME).until(
-           expected_conditions.presence_of_element_located((By.TAG_NAME, "h1"))
-        )
-        self.assertEqual('Django administration', header.text)
+        if self.is_django_5:
```

Comment:
Is it worthwhile wrapping these into a `wait_schedule_editor_ready()` method?

Reply:
This construct is only used in the tests that aren't allowed to access the schedule editor, so it would be `wait_admin_error'` or something, and is currently only used in 2 of the tests.

it may be worth wrapping anyway, but it's not a huge win,and should possibly be done with a look at some of the other  tests to consolidate some other bits of repeated logic.